tensorflow_datasets/scripts/documentation/generate_dataframe.py

```diff
@@ -0,0 +1,213 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Script which generates datasets dataframes.
+"""
+
+import functools
+import itertools
+import multiprocessing
+import os
+import imageio
+import tempfile
+import traceback
+from typing import List, Optional
+
+from absl import app
+from absl import flags
+from absl import logging
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow_datasets.core import lazy_imports_lib
+from tensorflow_datasets.core import features as features_lib
+from tensorflow_datasets.core.visualization import visualizer
+
+
+WORKER_COUNT_DATASETS = 10
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string(
+    'datasets', None,
+    'Comma separated list of datasets to generates. None for all datasets.')
+flags.DEFINE_string(
+    'dst_dir', tfds.core.gcs_path('visualization/dataframe'),
+    'Destination dir to save the dataframe html.')
+flags.DEFINE_boolean(
+    'overwrite', False, 'If True, overwrite the existing visualizations.')
+
+
+# pylint: disable=logging-format-interpolation,logging-not-lazy
+
+
+def _log_exception(fn):
```

Comment:
Rather than duplicating code, it would be cleaner to move those  functions into some shared `script_utils.py`

Reply:
Moved.